### ISSUE TITLE
[BUG] /proc/sys/net/bridge/bridge-nf-call-iptables: no such file or directory with centos7

### ISSUE THREAD

**Issue by hhstu** [openyurtio/openyurt#568]
**What happened**:
 yurtctl init  ...

err is:

```bash
[root@RJYF-T-210 ~]# ./yurtctl init --image-repository=registry.cn-hangzhou.aliyuncs.com/google_containers --kubernetes-version=v1.20.9   --pod-network-cidr=10.244.0.0/16
W1110 11:12:33.111889    7118 configset.go:202] WARNING: kubeadm cannot validate component configs for API groups [kubelet.config.k8s.io kubeproxy.config.k8s.io]
[init] Using Kubernetes version: v1.20.9
I1110 11:12:33.112060    7118 util.go:41] Setting ipv4 forward
I1110 11:12:33.112674    7118 util.go:50] Setting bridge settings for kubernetes.
Error: error execution phase initialize: Write file /proc/sys/net/bridge/bridge-nf-call-iptables fail: open /proc/sys/net/bridge/bridge-nf-call-iptables: no such file or directory


```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- OpenYurt version:
- Kubernetes version (use `kubectl version`):
- OS (e.g: `cat /etc/os-release`):

``` bash NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31" 
```
- Kernel (e.g. `uname -a`):
``` bash
[root@RJYF-T-210 ~]# uname  -a
Linux RJYF-T-210 3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```
- Install tools:
- Others:
- 

**others**

/kind bug

**Comment by rambohe-ch** [comment]
@hhstu Thank you for raising issue.

@zyjhtangtang would you like to reply this problem? 
/assign @zyjhtangtang

**Comment by rambohe-ch** [comment]
@hhstu the bug is fixed by pull request #578 
/close
